app/assets/javascripts/initializers/initializeTimeFixer.js

```diff
@@ -1,71 +1,61 @@
-function initializeTimeFixer() {
-  var utcTime = document.getElementsByClassName('utc-time');
-  var utcDate = document.getElementsByClassName('utc-date');
-  var utc = document.getElementsByClassName('utc');
+'use strict';
 
-  if (!utc) {
-    return;
-  }
+/* eslint-disable no-param-reassign */
 
-  function convertUtcTime(utcTime) {
-    var time = new Date(utcTime);
-    var options = {
-      hour: 'numeric',
-      minute: 'numeric',
-      timeZoneName: 'short'
-    };
-    time = new Intl.DateTimeFormat('en-US', options).format(time);
-    return time;
-  }
+function getFormat(options, formatValue) {
```

Comment:
`formatValue` should probably just be called `value` and the function could be called `formatDateTime` for example : )

Reply:
sounds good